.github/workflows/build-and-test.yml

```diff
@@ -11,10 +11,10 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v3
 
-      - name: Set up Node 16
+      - name: Set up Node 18
```

Comment:
If we're going to change this (which, by all means we should), can we move it to latest LTS (20... actually, pretty sure 22 was supposed to come out this week but maybe a bit too ahead of the curve there 😄 )

Reply:
Sure...I thought of it but I didn't 😬